DJHelper/DJHelper/Model/HelperExtensions.swift

```diff
@@ -20,8 +20,24 @@ extension Date {
 extension String {
     func dateFromString() -> Date? {
         let formatter = DateFormatter()
-        formatter.dateStyle = .medium
-        formatter.timeStyle = .medium
+        formatter.dateStyle = .short
         return formatter.date(from: self)
     }
 }
+
+extension UIView {
```

Comment:
Little animations like this are a creative and fun touch to enhance the user experience.